spring-boot-project/spring-boot/src/main/java/org/springframework/boot/web/server/ErrorPage.java

```diff
@@ -115,7 +115,7 @@ public boolean equals(Object obj) {
 		if (obj instanceof ErrorPage) {
 			ErrorPage other = (ErrorPage) obj;
 			boolean rtn = true;
-			rtn = rtn && ObjectUtils.nullSafeEquals(getExceptionName(), other.getExceptionName());
+			rtn = ObjectUtils.nullSafeEquals(getExceptionName(), other.getExceptionName());
```

Comment:
Thanks for the suggestion. Rather than assigning `rtn` on 117 and then again on line 118, the two could be combined:

```
boolean rtn = ObjectUtils.nullSafeEquals(getExceptionName(), other.getExceptionName());
```

Reply:
Thank you for your instruction. I have resubmitted it：
https://github.com/spring-projects/spring-boot/pull/22082/commits/9d2793e6699630af9a5f955cd282be8f2d19765e

Reply:
@wilkinsona Thank you for your instruction. I have resubmitted it：
https://github.com/spring-projects/spring-boot/pull/22082/commits/9d2793e6699630af9a5f955cd282be8f2d19765e